lib/decisionmaking-flow.js

```diff
@@ -199,7 +199,12 @@ INSERT DATA {
       dct:created ${sparqlEscapeDateTime(now)} ;
       ext:modified ${sparqlEscapeDateTime(now)} ;
       dct:type ${sparqlEscapeUri(subcaseType)} ;
-      ext:agendapuntType ${sparqlEscapeUri(agendaItemType)} .
+      ext:agendapuntType ${sparqlEscapeUri(agendaItemType)} ;
+      ext:bekrachtigdDoor ?mandatee .
```

Comment:
hmm, just thinking, we pass in `subcaseType`.
We only need `ext:bekrachtigdDoor` when it's a ratification right?
Can we realistically hit this code with another subcase type like with a resolution/motion?
Unsure of the possible scenarios